### DISCUSSION TITLE
save own bookmarks

### DISCUSSION THREAD

**Discussion by profectus** [dailydotdev/daily#406] [Feature Requests]
it would be good to be able to save your own bookmarks. Some analysis could be used to create a feed of interesting posts for others...

In general, I have a bit of a problem with bookmarks in browsers, the ability to rank and tag your own bookmarks would be nice.

**Comment by idoshamun** [comment]
Hi @profectus,

Thanks for the suggestion. Let me make sure that I understand your request correctly.
You would like to add custom links to your bookmarks and share them with other people. Is that right?

**Reply by JuanGRomeo** [reply]
Exactly, 

I think that's the point. Personally, since I have discovered daily.dev it's a nice source cross platform to read but also to manage my knowledge DB saving posts as bookmarks.

Usually, I struggle to manage other bookmarks betweenthe use of my work account and personal account on browsers. In the past I have also used other tools like Evernote or Notion.

But I think this app is the perfect tool for this, I think it would be very usefull if we can save whatever webpage we are visiting in our personal feed or Bookmark section, to have a personal space where we can find our favorites resources across all devices.

**Comment by GeoDaz** [comment]
As this extension takes the place of the new tab, I think it's important to be able to set your own bookmarks in the left-hand menu for easy access to github, for example. 
Like probably many developers, my favorites bar is full of folders, so having a shortcut list of favorites seems very interesting.

**Comment by idoshamun** [comment]
A neat workaround I can suggest for you in the meantime is to open private squads to manage these lists :)

**Reply by JuanGRomeo** [reply]
Ok I Have just tested this and works perfect for the moment. It requires some manual work to save all of my bookmarks one by one, but at least I have a place inside of daily.dev where I can access to them.

For the Future it would be nice to have a browser extension which do this for us.

I love the app, thanks!!
